.github/workflows/tests.yml

```diff
@@ -774,7 +774,11 @@ jobs:
           sudo chown $USER /var/lib/lxd/unix.socket
           export PATH="/home/runner/go/bin:$PATH"
           export CI=true
-          export DISABLE_VM_TESTS=true
+
+          # On 22.04, VMs do not work due to `virtiofsd` incompat.
+          if grep -qxF 'VERSION_ID="22.04"' /etc/os-release; then
```

Comment:
do we know if this is going to work or is it likely to break in GH runners?

Reply:
Oh sorry, I didn't read the context so I missed this was for the UI. I have not consulted with them and this has not been tested as this needs to be merged into `main` to have the needed bits for auth.

Should I drop that bit and ask them to test themselves?